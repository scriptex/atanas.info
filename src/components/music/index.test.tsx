import * as React from 'react';
import { render } from '@testing-library/react';

import { Music } from '.';

Object.defineProperty(global.window.HTMLMediaElement.prototype, 'load', {
	configurable: true,
	get() {
		return () => undefined;
	}
});

describe('Music component', () => {
	it('Should render the Music component', () => {
		const { asFragment } = render(<Music />);

		expect(asFragment()).toMatchSnapshot();
	});
});
