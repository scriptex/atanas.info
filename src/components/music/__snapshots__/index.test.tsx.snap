// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Music component Should render the Music component 1`] = `
<DocumentFragment>
  <main
    class="o-main o-main--high"
  >
    <section
      class="c-section c-section--music"
      id="music"
    >
      <div
        class="o-shell"
      >
        <div
          class="c-music"
        >
          <canvas
            height="768"
            id="canvas"
            width="1024"
          />
          <button
            class="c-music__menu"
            id="menu"
          >
            Pick a track
          </button>
          <button
            class="c-music__btn c-music__btn--pause"
            hidden=""
            id="pause"
          >
            Pause
          </button>
          <button
            class="c-music__btn c-music__btn--play"
            id="play"
          >
            Play
          </button>
          <audio
            controls=""
            crossorigin="anonymous"
            hidden=""
            id="audio"
            src="https://dl.dropbox.com/s/fh1igvfnzzucnmh/Scriptex_-_01_-_La_Playa%28Invader__2004%29.mp3?dl=0"
          />
          <div
            class="c-music__tracks"
            hidden=""
            id="tracks"
          >
            <button
              data-url="https://dl.dropbox.com/s/fh1igvfnzzucnmh/Scriptex_-_01_-_La_Playa%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - La Playa
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/p8o4hpat99va06b/Scriptex_-_02_-_Clash%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Clash
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/knvvlxhezgpiium/Scriptex_-_03_-_Joe_Cocker_-_U_Can_Leave_Your_Hat_On%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Joe Cocker - U Can Leave Your Hat On
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/ez9cj1v1vd6fs4m/Scriptex_-_04_-_Energy%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Energy
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/yzvcbrusy60j44g/Scriptex_-_05_-_Dance_Anthem%28Energy_part_II%29%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Dance Anthem (Energy part II)
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/uwf47rwdagnhu7n/Scriptex_-_06_-_Invader%28with_Serum%29%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex with Serum - Invader
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/pni6act68s5du8l/Scriptex_-_07_-_Some_Somtin_%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Some Somtin
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/agv3gkf885p6llf/Scriptex_-_08_-_Snap%21_-_The_Power%28Energy_mix%29%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Snap - The Power (Energy mix)
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/9qzi27kzuvs31km/Scriptex_-_09_-_Lights%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Lights
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/ape1sivb3nplb4j/Scriptex_-_10_-_Attack%28with_Serum%29%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex with Serum - Attack
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/udqrmqdd2tc58d8/Scriptex_-_11_-_Check_Yoself%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Check Yoself
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/trgmwdictt5e52q/Scriptex_-_12_-_Spheric_Triangle%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Spheric Triangle
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/0fij938hmssv8rc/Scriptex_-_13_-_Cycle_Round%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Cycle Round
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/5ik33i4uvtdiuj0/Scriptex_-_14_-_Get_This%21%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Get This!
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/l36265mpso0xcjr/Scriptex_-_15_-_Gabber_Myth%28Invader__2004%29.mp3?dl=0"
            >
              Scriptex - Gabber Myth
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/lg993y6522ph2qk/Scriptex_-_01_-_Energy%28Chemistry_EP__2005%29.mp3?dl=0"
            >
              Scriptex - Energy
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/cjuvrx9h20koi2c/Scriptex_-_02_-_Schwefelsaure%28Invader__2005%29.mp3?dl=0"
            >
              Scriptex - Schwefelsaure
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/lm41zk5xu5jwn3t/Scriptex_-_03_-_Cycle_Round%28Invader__2005%29.mp3?dl=0"
            >
              Scriptex - Cycle Round
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/n4vwf6504l89hif/Scriptex_-_01_-_Black_Soul%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - Black Soul
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/j69piqwo2f1a152/Scriptex_-_02_-_Gravity%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - Gravity
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/r9vsfqya8mmx03k/Scriptex_-_03_-_Winter_Dance%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - Winter Dance
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/q68qqq3xwcjxn1k/Scriptex_-_04_-_The_Heart%28edit%29%28with_Linderman%29%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex with Linderman - The Heart (edit)
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/tesdok7ses376ao/Scriptex_-_05_-_U_Got_Me%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - U Got Me
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/4ow68ywkysuqjrh/Scriptex_-_06_-_April_Joke%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - April Joke
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/dgljwawe10a0z42/Scriptex_-_07_-_Miracle%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - Miracle
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/9w1n4jyv92o8y4s/Scriptex_-_08_-_Spring_Or_Summer%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - Spring or Summer
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/ho49bfqya7dazbb/Scriptex_-_09_-_Last_Night_Preparation%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - Last Night Preparation
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/tvxs2hn8qeecj3k/Scriptex_-_10_-_U_Got_Me%28remix%29%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - U Got Me (remix)
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/02gnsu65idsoi4b/Scriptex_-_11_-_Tribute_To_Jay-Z%28with_Serum%29%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex with Serum - Tribute to Jay-Z
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/79axqq2wt5yjows/Scriptex_-_12_-_Accidently%28Dark__2006%29.mp3?dl=0"
            >
              Scriptex - Accidently
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/7zyeaqcedomaqo4/Scriptex___Serum_-_01_-_The_Driller%28Collection_Diamonds__2007%29.mp3?dl=0"
            >
              Scriptex & Serum - The Driller
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/lb12ikdua1lymd9/Scriptex___Serum_-_02_-_The_Blasters%28Collection_Diamonds__2007%29.mp3?dl=0"
            >
              Scriptex & Serum - The Blasters
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/wk2l5n5rlcaaff6/Scriptex___Serum_-_03_-_The_Demag%28Collection_Diamonds__2007%29.mp3?dl=0"
            >
              Scriptex & Serum - The Demag
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/ddttkfq93ulgkkg/Scriptex___Serum_-_04_-_The_Cat%28Collection_Diamonds__2007%29.mp3?dl=0"
            >
              Scriptex & Serum - The Cat
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/n1zu81og2ng0czn/Scriptex___Serum_-_05_-_The_Process_Plant%28Collection_Diamonds__2007%29.mp3?dl=0"
            >
              Scriptex & Serum - The Process Plant
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/1gui4oia31srxl3/Scriptex_-_Tamper_Trouble_EP%282007%29.mp3?dl=0"
            >
              Scriptex - Tamper Trouble
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/lzs7lkw2p37uor0/Scriptex_-_A1_-_Sadness__Tale%28Sadness_is_Allright_EP__2008%29.mp3?dl=0"
            >
              Scriptex - Sadness Tale
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/n5asr7nn1pnfybh/Scriptex_-_B1_-_Allright%28Sadness_is_Allright_EP__2008%29.mp3?dl=0"
            >
              Scriptex - Alright
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/qxdgioakm3dlct7/Scriptex%20-%2001%20-%20Alone.mp3?dl=0"
            >
              Scriptex - Alone
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/7lbaxr9nwrdhwq8/Scriptex%20-%2002%20-%20Movement.mp3?dl=0"
            >
              Scriptex - Movement
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/s13g4im31ffe38i/Scriptex%20-%2003%20-%20Drum%20Expo.mp3?dl=0"
            >
              Scriptex - Drum Expo
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/o2jmnrv18n3fw1m/Scriptex%20-%2004%20-%20One%20Night.mp3?dl=0"
            >
              Scriptex - One Night
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/u1kp8quzcfx7fca/Scriptex%20-%2005%20-%20The%20Winner.mp3?dl=0"
            >
              Scriptex - The Winner
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/g9v9f5299lkzln8/Scriptex%20-%2006%20-%20Brick_s%20Thoughts.mp3?dl=0"
            >
              Scriptex - Brick's Thoughts
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/r0du3qwlgbed5mh/Scriptex%20-%2007%20-%20Acceleration.mp3?dl=0"
            >
              Scriptex - Acceleration
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/4wmovo3qatd5tk9/Scriptex%20-%2008%20-%20Joy.mp3?dl=0"
            >
              Scriptex - Joy
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/jj2ccdke6nln7kw/Scriptex%20-%2009%20-%20Extraordinary.mp3?dl=0"
            >
              Scriptex - Extraordinary
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/fy14rpmgrhv7qk7/Scriptex%20-%20Party%20Session%20part%201.mp3?dl=0"
            >
              Scriptex - Party Session part 1
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/bxxp8xu6d45smsp/Scriptex%20-%20Party%20Session%20part%202.mp3?dl=0"
            >
              Scriptex - Party Session part 2
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/50ftjrrvbfv4muo/Scriptex%20-%20Party%20Session%20part%203.mp3?dl=0"
            >
              Scriptex - Party Session part 3
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/g0k385f5ud6et74/Scriptex%20-%20Party%20Session%20part%204.mp3?dl=0"
            >
              Scriptex - Party Session part 4
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/1w1yurfxv9r4zox/Scriptex%20-%2001%20-%20T.B.C..mp3?dl=0"
            >
              Scriptex - T.B.C.
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/pk8njjdlrfylxtg/Scriptex%20-%2002%20-%20Space.mp3?dl=0"
            >
              Scriptex - Space
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/pq1ux8rbzulyirm/Scriptex%20-%2003%20-%20Dixi.mp3?dl=0"
            >
              Scriptex - Dixi
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/d8ywpnnwmta2110/Scriptex%20-%2004%20-%20Andaverse.mp3?dl=0"
            >
              Scriptex - Andaverse
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/nz33i5z5ixuel4h/Scriptex%20-%2005%20-%20Military%20Situation.mp3?dl=0"
            >
              Scriptex - Military Situation
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/hfvjj04yv0v0avp/Scriptex%2001%20Trip-hop%20Is%20Dead.mp3?dl=0"
            >
              Scriptex - Trip Hop is Dead
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/fc825b0hen6h1y2/Scriptex%2002%20Bull%20Fight.mp3?dl=0"
            >
              Scriptex - Bull Fight
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/8tbc554e6ehdhkg/Scriptex%2003%20Curse.mp3?dl=0"
            >
              Scriptex - Curse
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/21rz84t9jcjkesc/Scriptex%2004%20Reload.mp3?dl=0"
            >
              Scriptex - Reload
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/xrr53iinlkrdb7r/Scriptex%2005%20Alien%20Attack.mp3?dl=0"
            >
              Scriptex - Alien Attack
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/d0peqokyccskarx/Scriptex%2006%20The%20Professionalist%28question%20%E2%84%9613%29.mp3?dl=0"
            >
              Scriptex - The Professionalist
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/l80146rff18q7a2/Scriptex%2007%20Mesmerizing.mp3?dl=0"
            >
              Scriptex - Mesmerizing
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/oxb5kcvgymz7lm1/Scriptex%2008%20Robotica.mp3?dl=0"
            >
              Scriptex - Robotica
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/n6m3ip4a9isk2rp/Scriptex%2009%20Boom%20x%203%28Question%20%E2%84%9614%29.mp3?dl=0"
            >
              Scriptex - Boom x3 (Question No. 14)
            </button>
            

            <button
              data-url="https://dl.dropbox.com/s/oebgh2r7p55yp2a/Scriptex%2010%20Alien%20Attack%28Full%20Version%29.mp3?dl=0"
            >
              Scriptex - Alien Attack (Full version)
            </button>
          </div>
          <h2>
            Pick a track and 
            <br />
             press the play button.
          </h2>
        </div>
        <div
          class="c-section__actions"
        >
          <button
            class="c-btn "
            type="button"
          >
            Get in touch
          </button>
          <ul
            class="c-section__actions-list"
          >
            <li>
              <button
                class="c-btn c-section__actions-close"
                type="button"
              >
                Close
              </button>
            </li>
            <li>
              <button
                class="c-btn "
                type="button"
              >
                <svg
                  class="c-section__actions-icon"
                >
                  <use
                    xlink:href="#svg-email"
                  />
                </svg>
                <span>
                  Contact me
                </span>
                <strong>
                  directly
                </strong>
              </button>
            </li>
            <li>
              <a
                class="c-btn"
                href="https://app.usebraintrust.com/talent/782/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <svg
                  class="c-section__actions-icon"
                >
                  <use
                    xlink:href="#svg-braintrust"
                  />
                </svg>
                <span>
                  Hire me on
                </span>
                <strong>
                  Braintrust
                </strong>
              </a>
            </li>
            <li>
              <a
                class="c-btn"
                href="https://www.toptal.com/resume/atanas-atanasov"
                rel="noopener noreferrer"
                target="_blank"
              >
                <svg
                  class="c-section__actions-icon"
                >
                  <use
                    xlink:href="#svg-toptal"
                  />
                </svg>
                <span>
                  Hire me on
                </span>
                <strong>
                  Toptal
                </strong>
              </a>
            </li>
          </ul>
        </div>
      </div>
    </section>
  </main>
</DocumentFragment>
`;
